controllers/common/views/information-page.njk

```diff
@@ -41,6 +47,16 @@
                     {% endfor %}
                 </div>
             {% endif %}
+
+            {# Flexible content #}
+            {% if content.flexibleContent %}
+                {% for part in content.flexibleContent %}
```

Comment:
This feels a little awkward here that each flexible content part is a _new_ content area, the way we use this elsewhere is that flexible content is all within a _single_ content block.